packages/webawesome/src/components/dropdown/dropdown.ts

```diff
@@ -227,6 +239,12 @@ export default class WaDropdown extends WebAwesomeElement {
       return;
     }
 
+    // if this dropdown is already open, do nothing
+    // (this can happen when wa-hide was cancelled)
+    if (this.popup.active) {
+      return;
+    }
```

Comment:
I don't love checking `this.popup.active` directly. I'm happy to use alternative properties or checks here, but this seemed the most direct, and simpliest way to validate if the menu was already opened.